<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-240</x>
      <y>-50</y>
      <width>1081</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 4, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>20</width>
       <height>441</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777212</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>72</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>39</y>
       <width>311</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 68, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="imag_cdr">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>37</y>
        <width>244</width>
        <height>244</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>244</width>
        <height>244</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>551</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 68, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>381</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 32, 3);
font: 22pt &quot;MS Shell Dlg 2&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Probabilité:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Prédiction:</string>
       </property>
      </widget>
      <widget class="QLabel" name="pred_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>-----------</string>
       </property>
      </widget>
      <widget class="QLabel" name="proba_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>-----------</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Les résultats</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="save_btn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>380</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 68, 255);
border-radius:5px;
padding-top:5px;
color:rgb(226,234,216);
font: 25 14pt &quot;Yu Gothic Light&quot;;

}
QPushButton:hover{
background-color: rgb(0, 85, 255);
}
QPushButton:pressed{
background-color:rgb(130,170,90);
}</string>
     </property>
     <property name="text">
      <string>Sauvegarde</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>390</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="importbtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>410</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 68, 255);
border-radius:5px;
padding-top:5px;
color:rgb(226,234,216);
font: 25 14pt &quot;Yu Gothic Light&quot;;

}
QPushButton:hover{
background-color: rgb(0, 85, 255);
}
QPushButton:pressed{
background-color:rgb(130,170,90);
}</string>
     </property>
     <property name="text">
      <string>Charger une image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="preditct_btn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>430</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 68, 255);
border-radius:5px;
padding-top:5px;
color:rgb(226,234,216);
font: 25 14pt &quot;Yu Gothic Light&quot;;

}
QPushButton:hover{
background-color: rgb(0, 85, 255);
}
QPushButton:pressed{
background-color:rgb(130,170,90);
}</string>
     </property>
     <property name="text">
      <string>Predict</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>330</y>
       <width>701</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <column>
      <property name="text">
       <string>ID Test</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Client</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom Prenon </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CIN Client</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path chest x-ray </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Resultat</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date de Test</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
